vscode-extension/package.json

```diff
@@ -13,10 +13,72 @@
     "Other"
   ],
   "activationEvents": [
-    "workspaceContains:./devbox.json"
+    "onStartupFinished"
   ],
   "main": "./out/extension.js",
-  "contributes": {},
+  "contributes": {
+    "commands": [
+      {
+        "command": "devbox.setupDevContainer",
+        "title": "Devbox: Generate Dev Containers config files"
+      },
+      {
+        "command": "devbox.add",
+        "title": "Devbox: Add - add packages to your devbox project"
+      },
+      {
+        "command": "devbox.shell",
+        "title": "Devbox: Shell - Go to devbox shell in the terminal"
+      },
+      {
+        "command": "devbox.install",
+        "title": "Devbox: Install - Install packages specified in devbox.json"
+      },
+      {
+        "command": "devbox.init",
+        "title": "Devbox: Init - Initiate a devbox project"
```

Comment:
Can we add `devbox run` command when it is out? (which should be tomorrow?)

Reply:
Just added it in the last commit